### PR TITLE
Lvnod 27 create user model

### PR DESCRIPTION
**Pull request by yarvanya**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-27)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-43)

## Code reviewers

- [x] @yurii-kokhalevych
- [x] @AlbaMontis
- [x] @shyyka

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR

### CODE REVIEW

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

File: server/models/user.model.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    password: {
+      type: DataTypes.TEXT,
```
Comment (AlbaMontis) [inline server/models/user.model.js]: What about type: DataTypes.VIRTUAL?

File: .jscsrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "preset": "google",
+  "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties",
```
Comment (AlbaMontis) [inline .jscsrc]: "ignoreProperties" -Deprecated, Please use the Object value

File: .jscsrc [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  "maximumLineLength": 100
+}
```
Comment (AlbaMontis) [inline .jscsrc]: Suggestion: set some "validateIndentation".

**Review by AlbaMontis — APPROVED** [review]

**Review by yurii-kokhalevych — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

File: server/models/user.model.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('user', {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false
```
Comment (4asof4ik) [inline server/models/user.model.js:12]: validation

File: server/models/user.model.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    password: {
+      type: DataTypes.VIRTUAL,
```
Comment (4asof4ik) [inline server/models/user.model.js:15]: Virtual??
validation

File: server/models/user.model.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('user', {
```
Comment (4asof4ik) [inline server/models/user.model.js:4]: User but not user

File: server/migrations/20170520152757-create_table_user.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('users', {
```
Comment (4asof4ik) [inline server/migrations/20170520152757-create_table_user.js:5]: Users

File: server/migrations/20170520152757-create_table_user.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+        allowNull: false
+      },
+      status_state: {
+        type: Sequelize.STRING,
+        allowNull: false
+      }
+    });
+  },
+  down: function(queryInterface, Sequelize) {
+    return queryInterface.dropTable('users');
```
Comment (4asof4ik) [inline server/migrations/20170520152757-create_table_user.js:33]: Users

File: server/seeders/user.seed.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.bulkInsert('users', [
```
Comment (4asof4ik) [inline server/seeders/user.seed.js:5]: Users

File: server/seeders/user.seed.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+        password: '13452',
+        profile_id: 2,
+        is_activate: true,
+        status_state: ''
+      }
+    ], {});
+  },
+  down: function(queryInterface, Sequelize) {
+    return queryInterface.bulkDelete('users', null, {});
+  }
```
Comment (4asof4ik) [inline server/seeders/user.seed.js:24]: Users

File: server/models/user.model.js [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    password: {
+      type: DataTypes.VIRTUAL,
+      allowNull: false
+    },
+    profile_id: {
+      type: DataTypes.INTEGER
```
Comment (4asof4ik) [inline server/models/user.model.js:19]: Check  how build relationship in tutorial which l send early.

**Pull request closed without merging**